purchase_request_to_requisition/wizard/purchase_request_line_make_purchase_requisition.py

```diff
@@ -77,8 +77,6 @@ def _get_requisition_line_search_domain(self, requisition, item):
                  item.product_uom_id.id or False),
                 ('account_analytic_id', '=',
                  item.line_id.analytic_account_id.id or False)]
-        if not item.product_id:
-            vals['name'] = item.name
```

Comment:
Instead of removing, why not do:
'if not item.product_id:'
'vals.append(('name', '=', item.name))'

As you're correctly proposing in purchase_request_to_rfq/wizard/purchase_request_line_make_purchase_order.py